bootstrap.sh

```diff
@@ -64,3 +64,21 @@ do
     && cp -n Local.templates/$CONFIG_FILE Local/$CONFIG_FILE \
   )
 done
+
+# Build Yubikit
+YUBIKIT_DIR=Carthage/Checkouts/yubikit-ios
+YUBIKIT_OUT=ThirdParty/YubiKit
+SRCDIR=$PWD
+
+rm -rf $YUBIKIT_DIR
+rm -rf $YUBIKIT_OUT
+
+git clone -b simulator_release --single-branch --depth 1 https://github.com/jumde/yubikit-ios/ $YUBIKIT_DIR
+
+mkdir -p $SRCDIR/$YUBIKIT_OUT/include
+pushd $YUBIKIT_DIR/
+git checkout simulator_release
```

Comment:
This is no longer required, since you cloned only this branch :)

Reply:
yes, I missed this earlier. Should be fixed now.